app/index.html

```diff
@@ -14,7 +14,6 @@
   <script src="script/index.js"></script>
 </head>
 <body>
```

Comment:
This file doesn't seem right.  It's been submitted with merge conflicts, but here you've removed the markers for the conflict but left in place both versions of the html.  Normally, you would remove one or the other